collector/pg_database_test.go

```diff
@@ -15,13 +15,80 @@ package collector
 import (
 	"context"
 	"testing"
+	"time"
 
 	"github.com/DATA-DOG/go-sqlmock"
 	"github.com/prometheus/client_golang/prometheus"
+
```

Comment:
I'm not sure what this extra space is for.  I'm surprised that ci isn't complaining. We can remove it. 
```suggestion
```